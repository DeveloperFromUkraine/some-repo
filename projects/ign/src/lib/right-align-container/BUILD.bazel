load("@angular//:index.bzl", "ng_module")
load("@io_bazel_rules_sass//:defs.bzl", "sass_binary")

package(default_visibility = ["//visibility:public"])

sass_binary(
    name = "style",
    src = "right-align-container.scss",
)

ng_module(
    name = "right-align-container-module",
    srcs = [
        "right-align-container.module.ts",
        "right-align-container.component.ts",
    ],
    assets = [
        ":right-align-container.css",
        "right-align-container.html"
        ],
    tsconfig = "//:tsconfig.json",
    deps = [
        "@npm//tslib",
        "@npm//@types",
        "@angular//packages/core",
        "@angular//packages/common",
    ]
)
