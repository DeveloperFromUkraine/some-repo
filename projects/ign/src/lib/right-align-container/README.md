## Use
  This container will align options (such as expandable search, vertical menu, and others) towards the right-end of its space

  ### Content
    - The right align container can contain a variety of elements, including buttons, images, and text.


## API
    ### Selector
      * ign-right-align-container

    ### Exported As
      * RightAlignContainerComponent


## Implementation HTML
  <ign-right-align-container>This text is right aligned.</ign-right-align-container>


## Implementation TS
    None