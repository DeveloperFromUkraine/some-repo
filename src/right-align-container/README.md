# Options Container

This container will align options (such as expandable search, vertical menu, and others) towards the right-end of its space

## API
    ### Selector
      * ign-right-align-container

    ### Exported As
      * RightAlignContainerComponent

## Implementation HTML
  <ign-right-align-container>This text is right aligned.</ign-right-align-container>

## Implementation TS
    None