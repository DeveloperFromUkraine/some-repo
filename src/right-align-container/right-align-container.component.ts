import { Component } from '@angular/core';

@Component({
    selector: 'ign-right-align-container',
    templateUrl: './right-align-container.html',
    styleUrls: ['./right-align-container.scss'],
})
export class RightAlignContainerComponent {}
