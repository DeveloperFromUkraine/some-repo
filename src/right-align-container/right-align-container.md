*** HTML *** 
# Right Align Container

## Use
This container will align options (such as expandable search, vertical menu, and others) towards the right-end of its space

## Sample
<mat-tab-group>
    <mat-tab label="Component Sample"><div class="tab-height">
        <ign-right-align-container>This text is right aligned.</ign-right-align-container>
    </div></mat-tab>
    <mat-tab label="HTML"><div class="tab-height">
        <table style="width:100%">
            &lt;ign-right-align-container&gt;This text is right aligned.&lt;/ign-right-align-container&gt;
         </table></div>
    </mat-tab>
</mat-tab-group>

## Content
* The right align container can contain a variety of elements, including buttons, images, and text.


## Style
* The right align container is already styled and does not require further styling.

*** TypeScript *** 

*** ExternalTypeScript ***

*** Imports ***