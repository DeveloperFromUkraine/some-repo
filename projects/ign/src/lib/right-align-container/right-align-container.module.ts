import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';

import { RightAlignContainerComponent } from './right-align-container.component';

@NgModule({
    imports: [CommonModule],
    exports: [RightAlignContainerComponent],
    declarations: [RightAlignContainerComponent]
})
export class RightAlignContainerModule {

}
