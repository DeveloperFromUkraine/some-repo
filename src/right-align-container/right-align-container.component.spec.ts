import { ComponentFixture, TestBed } from '@angular/core/testing';
import { RightAlignContainerComponent } from './right-align-container.component';
import { ComponentTest } from '../../test/test-bed/component';

describe('Right Align Container', () => {
  let fixture: ComponentFixture<RightAlignContainerComponent>;

  beforeEach(async () => {
    await ComponentTest.createTestBed([], [RightAlignContainerComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(RightAlignContainerComponent);
  });

  it('Should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
