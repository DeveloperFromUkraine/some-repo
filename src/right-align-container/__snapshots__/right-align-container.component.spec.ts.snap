// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Right Align Container Component Should match snapshot 1`] = `
<ign-right-align-container>
  
</ign-right-align-container>
`;
